### PR TITLE
O0484 piyush web

### PR DESCRIPTION
**Pull request by rahul1choudhary**

### CODE REVIEW

File: components/signup/stepTwo.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ class StepTwo extends React.Component {
                                                                 false
                                                             )
                                                         }
-                                                        className="col-primary c-fw-600 p-3 cursor-pointer"
+                                                        className="col-primary c-fw-600 p-3 cursor-pointer text-underline"
```
Comment (rakeshshrivas) [inline components/signup/stepTwo.js:109]: Rename class Name to `text-hover-underline`

File: styles/_sizes.scss [hunk 2]
```diff
@@ -34,13 +34,15 @@
         word-spacing: 8px;
     }
 }
-.c-fs-6{
-    font-size: var(--c-fs-6)!important;
-    @media(max-width:425px) {
-        word-spacing:8px;      
-    }
+.c-fs-6 {
```
Comment (rakeshshrivas) [inline styles/_sizes.scss:37]: Revert old changes

File: styles/_var.scss [hunk 3]
```diff
@@ -1,12 +1,12 @@
 @import url('https://fonts.googleapis.com/css2?family=Libre+Baskerville:wght@400;700&display=swap');
-@import url('https://fonts.googleapis.com/css2?family=Plus+Jakarta+Sans:wght@200;300;400;500;600;700;800&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap');
```
Comment (rakeshshrivas) [inline styles/_var.scss:2]: revert old font lib also add new font

File: styles/_var.scss [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
 // font-family: 'Plus Jakarta Sans', sans-serif;
 // font-family: 'Libre Baskerville', serif;
 
 :root{
     // font family 
     --c-head:'Libre Baskerville', serif;
-    --c-body:'Plus Jakarta Sans', sans-serif;
+    --c-body:'Inter', sans-serif;
```
Comment (rakeshshrivas) [inline styles/_var.scss:9]: Correct this

**Review by rakeshshrivas — CHANGES_REQUESTED** [review]

File: components/signup/stepTwo.js (hunk 1 above)
Comment (nisha-walkover) [reply components/signup/stepTwo.js]: done

File: styles/_sizes.scss (hunk 2 above)
Comment (nisha-walkover) [reply styles/_sizes.scss:40]: done

File: styles/_var.scss (hunk 3 above)
Comment (nisha-walkover) [reply styles/_var.scss:3]: done

File: styles/_var.scss (hunk 4 above)
Comment (nisha-walkover) [reply styles/_var.scss]: done

**Review by rakeshshrivas — APPROVED** [review]

**Review by rakeshshrivas — APPROVED** [review]

File: styles/globals.scss [hunk 5]
```diff
@@ -8,7 +8,7 @@ next-route-announcer{
   }
 }
 body {
-  font-size: 16px !important;
+  font-size: 14px !important;
```
[the remark this answers is not in the document]
Comment (rakeshshrivas) [reply styles/globals.scss]: Done

**Pull request merged**